homeassistant/components/mochad/light.py

```diff
@@ -15,13 +15,19 @@
     LightEntity,
 )
 from homeassistant.const import CONF_ADDRESS, CONF_DEVICES, CONF_NAME, CONF_PLATFORM
-from homeassistant.core import HomeAssistant
 from homeassistant.helpers import config_validation as cv
-from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
 from . import CONF_COMM_TYPE, DOMAIN, REQ_LOCK
 
+if TYPE_CHECKING:
```

Comment:
At this point, we are unsure if this is helpful, so we should not do this until that has been decided on (or, there are situation-specific performance needs for it).